settings.json

```diff
@@ -1,7 +1,7 @@
 {
-    "oneVizionUrl" : "",
-    "oneVizionLogin" : "",
-    "oneVizionPwd" : "",
+    "ovUrl" : "",
```

Comment:
What about backward compatibility? Nobody uses 'master' as version for sms-notification?

Reply:
Good question. Ideally, no client should use the master version.
A year ago, a script was executed to switch to version 1.0
@asmoiseenko, what do you think?

Reply:
I saw some installs are on master, @Anastasiyaaa, can you provide SQL script I can run across all installs to check where master is installed? Then I can just upgrade them to the latest tag version and we can merge these changes. I think we will need to create a new version branch - 2.X

Reply:
Script for searching sms modules master version attached:
[select_master_sms_modules.txt](https://github.com/ov-integrations/sms-notification/files/13696849/select_master_sms_modules.txt)

Also attached is a script for upgrading sms modules of the master version to version 1.1 (latest):
[upgrade_sms_modules_to_1.1.txt](https://github.com/ov-integrations/sms-notification/files/13696884/upgrade_sms_modules_to_1.1.txt)

2.X branch created.

Reply:
Only a few uat installs had master version installed, and it was disabled anyway. I've updated those and we can move forward now